host-components/src/keep-manager-tester.rs

```diff
@@ -0,0 +1,172 @@
+extern crate reqwest;
+#[macro_use]
+extern crate serde_derive;
+
+use ::host_components::*;
+use reqwest::blocking::Response;
+use std::collections::HashMap;
+//TODO - better user input
+use std::io;
+
+fn main() {
+    let mut user_input = String::new();
+
+    let mut command1: HashMap<String, String> = HashMap::new();
+    command1.insert("command".to_string(), "list-all".to_string());
+    let mut command2: HashMap<String, String> = HashMap::new();
+    command2.insert("command".to_string(), "new-keep".to_string());
+    command2.insert("keep-arch".to_string(), "wasi".to_string());
+    command2.insert("auth-token".to_string(), "a3f9cb07".to_string());
+    let mut command3: HashMap<String, String> = HashMap::new();
+    command3.insert("command".to_string(), "list-keeps".to_string());
```

Comment:
for readability `command_list_all` `command_new_keep` `command_list_keeps` would have been nice.

Reply:
You're not wrong.  I was in busy cut and paste mode.  Fixed.